vowpalwabbit/parser/flatbuffer/parse_flat_example.cc

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) by respective owners including Yahoo!, Microsoft, and
```

Comment:
So far we have `parse_example.h` and `parse_example_json.h`. Maybe rename this to `parse_example_flatbuffer` for consistency?

Reply:
Changed